### PR TITLE
Materials: Use athena data

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/webapi/profiles/ingredient/IngredientInfo.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        if (materialName == null) {
+            return new ItemStack(Items.AIR);
+        }
+
+        Optional<Item> item = Registry.ITEM.getOptional(new ResourceLocation(materialName));
+        ItemStack itemStack = item.map(ItemStack::new).orElseGet(() -> new ItemStack(Items.AIR));
+
+        if (metadata != null) {
+            itemStack.setDamageValue(Integer.parseInt(metadata));
+        }
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/core/webapi/profiles/ingredient/IngredientInfo.java:37]: How does this support skulld?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/webapi/profiles/ingredient/IngredientInfo.java:37]: Oh right. Oops

File: common/src/main/java/com/wynntils/core/webapi/WebManager.java [hunk 2]
```diff
@@ -226,42 +221,6 @@ private static void tryLoadItemGuesses() {
         // Check for success
     }
 
-    private static void tryLoadItemMaterialIdMap() {
```
Comment (magicus) [inline common/src/main/java/com/wynntils/core/webapi/WebManager.java:229]: How do we get the data if you remove the json download? Not sure I'm following the changes here.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java:229]: I made scyu update athena so the new ids are added there, and will be always up-to-date.

Comment (magicus) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java:229]: So prior to this fix we got info from two sources, old ids from Athena and new from the json file? And now all are merged into one in Athena?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java:229]: Yes

**Review by magicus — APPROVED** [review]

**Pull request merged**